src/Controls/src/Core/Controls.Core.csproj

```diff
@@ -7,6 +7,8 @@
     <IsPackable>false</IsPackable>
     <_MauiDesignDllBuild Condition=" '$(OS)' != 'Unix' And '$(MSBuildRuntimeType)' == 'Full'">True</_MauiDesignDllBuild>
     <GitInfoReportImportance>high</GitInfoReportImportance>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
What is this warning? Missing docs?

Reply:
Yes, exactly: Compiler Warning (level 4) CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591